### PR TITLE
A few JSDoc fixes.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: lib/fetchResource.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const upgradeToHttps = require('./upgradeToHttps');
 /**
  * Download the resource
  *
- * @param {string} resourceUrl
+ * @param {string} resourceUrl The URL to fetch
```
Comment (mozfreddyb) [inline lib/fetchResource.js]: Looks good, but note how other JSDoc comments seperate the variable name and the description with a dash (`-`)

File: lib/moduleForProtocol.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const url = require('url');
 /**
  * Choose corresponding module (http or https) for downloading a resource.
  *
- * @param {string} urlString
+ * @param {string} urlString Input URL
```
Comment (mozfreddyb) [inline lib/moduleForProtocol.js]: please add a `-`

File: lib/shuffleArray.js [hunk 3]
```diff
@@ -8,6 +8,9 @@
  * Shuffle the elements of the array.
  *
  * https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
+ *
+ * @param {Array} oldArray The input Array
+ * @return {Array} The new shuffled Array
```
Comment (mozfreddyb) [inline lib/shuffleArray.js]: please add a `-` for the 2 lines above

File: lib/upgradeToHttps.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ const secureHosts = require('./data/secureHosts');
 /**
  * Upgrade the given URL to HTTPS, if it's from a host believed to support it.
  *
- * @param {string} urlString
+ * @param {string} urlString Input URL
```
Comment (mozfreddyb) [inline lib/upgradeToHttps.js]: please add a `-`

**Comment by XhmikosR** [comment]
@mozfreddyb should be OK now

**Pull request merged**